sdk/python/kfp/v2/v2_namespace_alias_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 The Kubeflow Authors
```

Comment:
Feels a bit odds to have a test file hanging out in the v2 folder. How about merging it into https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/compiler/compiler_test.py
WDYT?

Reply:
You're right that neither follows the convention. Putting this in compiler_test.py might be cleaner, but I don't have a strong opinion on this. So up to you, I'm fine keeping it here :)